src/utils/symbolGenerators.ts

```diff
@@ -52,14 +52,24 @@ module Plottable {
           case "triangle-down":
             sizeFactor = Math.sqrt(3);
             break;
+          default:
+            sizeFactor = 1;
+            break;
         }
 
         return sizeFactor * Math.pow(SYMBOL_GENERATOR_RADIUS, 2);
       };
 
+      function ensureSymbolType(symTypeString: string) {
```

Comment:
It's specific to d3.  We could make it static, but it's mainly only useful for this single symbolGeneartor